src/test/java/uk/gov/homeoffice/digital/sas/timecard/producers/KafkaProducerServiceTest.java

```diff
@@ -36,10 +48,10 @@ class KafkaProducerServiceTest {
   private KafkaProducerService<TimeEntry> kafkaProducerService;
 
   @Mock
-  private ListenableFuture<SendResult<String, KafkaEventMessage<TimeEntry>>> responseFuture;
+  private CompletableFuture<SendResult<String, KafkaEventMessage<TimeEntry>>> responseFuture;
 
   @ParameterizedTest
-  @EnumSource(value = KafkaAction.class, names = {"CREATE", "UPDATE"})
+  @EnumSource(value = KafkaAction.class, names = {"CREATE", "UPDATE", "DELETE"})
   void sendMessage_actionOnResource_messageIsSentWithCorrectArguments(KafkaAction action) {
```

Comment:
nit: made a similar comment on another PR. Might be better to not provide the `names` param, and let the test run against all enum names to auto cover future additions.

Reply:
Actioned